back-end/API/app/controllers/auth_controller.py

```
@@ -0,0 +1,95 @@
+    user = user_service.update_password(current_user, dto)
+    return user.model_dump(), 200
+
+@auth_blueprint.route('/resetPassword', methods=['POST'])
+def resetPassword():
+    try:
+        data = request.get_json()
+        decryptedData = json.loads(decrypt(data['token']))
+        if data['password'] == data['confirmPassword']:
+            try:
+                reset_date = float(decryptedData['resetDate'])
+                if (reset_date + 900) > datetime.now().timestamp():
```

Ruff (0.15.13)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)